cpp/dolfinx/fem/FunctionSpace.h

```diff
@@ -37,6 +38,10 @@ class FunctionSpace
   FunctionSpace(std::shared_ptr<const mesh::Mesh> mesh,
                 std::shared_ptr<const fem::FiniteElement> element,
                 std::shared_ptr<const fem::DofMap> dofmap);
+  
+  FunctionSpace(std::shared_ptr<const mesh::MeshView> meshView,
```

Comment:
Maybe we should call it `mesh_view` when it is a variable, as it is closer to how we define other variables (such as mesh, element and dofmap)